app/src/main/java/com/github/se/project/MainActivity.kt

```diff
@@ -1,5 +1,6 @@
 package com.github.se.project
 
+import CreateTutorSchedule
```

Comment:
the import should be sometihnig like import com.github.se.project.ui.profile.CreateTutorSchedule